polaris.shopify.com/content/patterns/date-picking.ts

```diff
@@ -435,387 +474,501 @@ const pattern: MultiVariantPattern = {
             <Popover
-            active={popoverActive}
-            autofocusTarget="none"
-            preferredAlignment="left"
-            preferredPosition="below"
-            fluidContent
-            fullHeight
-            activator={
-              <Button
-                icon={CalendarMinor}
-                onClick={() => setPopoverActive(!popoverActive)}
-              >
-                {buttonValue}
-              </Button>
-            }
-            onClose={() => setPopoverActive(false)}
-          >
-            <Box padding={{ xs: 2 }}>
-              <Columns columns={{ xs: "1fr", md: "1fr 4fr" }} ref={datePickerRef}>
-                <OptionList
-                  options={ranges.map((range) => ({
-                    value: range.alias,
-                    label: range.title,
-                  }))}
-                  selected={activeDateRange.alias}
-                  onChange={(value) => {
-                    setActiveDateRange(
-                      ranges.find((range) => range.alias === value[0])
-                    );
+              active={popoverActive}
+              autofocusTarget="none"
+              preferredAlignment="left"
+              preferredPosition="below"
+              fluidContent
+              sectioned={false}
+              fullHeight
+              activator={
+                <Button
+                  size="slim"
+                  icon={CalendarMinor}
+                  onClick={() => setPopoverActive(!popoverActive)}
+                >
+                  {buttonValue}
+                </Button>
+              }
+              onClose={() => setPopoverActive(false)}
+            >
+              {/* <Box padding={{ xs: 2 }} maxWidth={"728px"}> */}
+              <Popover.Pane fixed>
+                <Columns
+                  columns={{
+                    xs: "1fr",
+                    mdDown: "1fr",
+                    md: "max-content max-content",
                   }}
-                />
-                <Box padding={{ md: 6 }}>
-                  <AlphaStack fullWidth>
-                    <Inline>
-                      <div style={{ flexGrow: 1 }}>
-                        <TextField
-                          role="combobox"
-                          label={"Since"}
-                          labelHidden
-                          prefix={<Icon source={CalendarMinor} />}
-                          value={inputValues.since}
-                          onChange={handleStartInputValueChange}
-                          onBlur={handleInputBlur}
-                          autoComplete="off"
+                  gap={0}
+                  ref={datePickerRef}
+                >
+                  <Box
+                    maxWidth={mdDown ? "516px" : "212px"}
+                    width={mdDown ? "100%" : "212px"}
+                    padding={{ xs: 5, md: 0 }}
+                    paddingBlockEnd={{ xs: 1, md: 0 }}
+                  >
+                    {mdDown ? (
```

Comment:
I don't see anything in the pattern docs about altering the layout when on a small screen - is this an opinion the pattern cares about and should be enforced?

Why is this a media query? What if the screen is wide, but the content column is thin? Wouldn't we want a similar behaviour (eg; if the date range picker was in a side-bar)? How do we handle that?

Reply:
Agreed, at this point the onus is on replicating what's in admin. This is a symptom of this decision. The decision itself is what should be resolved. Should it be resolved post or pre mvp is an open question however